packages/jwt-audit-analytics-writer/src/index.ts

```diff
@@ -37,7 +38,10 @@ await retryConnection(
   dbContext,
   config,
   async (db) => {
-    await setupDbServiceBuilder(db.conn).setupStagingTables();
+    await setupDbServiceBuilder(db.conn, config).setupStagingTables([
+      JwtDbTable.client_assertion,
+      JwtDbTable.generated_token,
```

Comment:
```suggestion
      JwtDbTable.generated_token,
      JwtDbTable.client_assertion,
```

It's not a real problem, but let's keep order right anyway, even if there aren't any enforcements on the temporary ones, just to stay consistent with the logic, since the foreign key of client_assertion_audit depends on generated_token